package-testing/dotnet-sdk-relay/EppoSDKRelay/Startup.cs

```diff
@@ -0,0 +1,63 @@
+using eppo_sdk;
+using Microsoft.OpenApi.Models;
+
+namespace EppoSDKRelay;
+
+public class Startup
+{
+    static readonly String apiHost = Environment.GetEnvironmentVariable("EPPO_API_HOST") ?? "localhost";
+    static readonly String apiPort = Environment.GetEnvironmentVariable("EPPO_API_PORT") ?? "5000";
+    static readonly String apiToken = Environment.GetEnvironmentVariable("EPPO_API_TOKEN") ?? "NO_TOKEN";
+
+
+    public static void InitEppoClient()
+    {
+        var url =  "http://" + apiHost + ":" + apiPort;
+        Console.WriteLine("Initializating SDK pointed at" + url);
+
+        var eppoClientConfig = new EppoClientConfig(apiToken, new AssignmentLogger())
```

Comment:
should we also be disabling graceful mode somewhere around here?

Reply:
need to add it first
[FF-2510](https://linear.app/eppo/issue/FF-2510/graceful-mode-in-dotnet-sdk)